docs/getting-started/install_turtles_operator.md

```diff
@@ -4,4 +4,17 @@ sidebar_position: 4
 
 # Install Rancher Turtles Operator
 
-This section needs to be filled out.
\ No newline at end of file
+Rancher turtles requires to have a direct connection to the Rancher Manager cluster. This could be achieved by:
+1. Installing in in the same cluster as the Rancher Manager.
+
```

Comment:
We can add a note to say in the future we may support different deployment toplogies.